### DISCUSSION TITLE
GitHub Copilot Extension is not available in Visual Studio 2022.

### DISCUSSION THREAD

**Discussion by saashfaque** [community/community#110961] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

GitHub Copilot Extension is not available in Visual Studio 2022. Can you please help me on this? Do we need to install the extension separately?

**Comment by davecheney** [comment, marked as answer]
@saashfaque you'll find the installation instructions here https://learn.microsoft.com/en-us/visualstudio/ide/visual-studio-github-copilot-extension?view=vs-2022
